wg-batch/charter.md

```diff
@@ -0,0 +1,66 @@
+# WG Batch Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+Discuss and enhance the support for Batch (eg. HPC, AI/ML, data analytics) workloads
+in core Kubernetes. We want to unify the way users deploy batch workloads to improve
+portability and to simplify supportability for Kubernetes providers.
+
+### In scope
+
+- Additions to the batch API group, currently including Job and CronJob resources
+  that benefit batch use cases such as HPC, AI/ML and data analytics.
+- Primitives for job-level queueing, not limited to the k8s Job resource.
+- Primitives to control and maximize utilization of resources in fixed-size clusters (on-prem) and elastic clusters (cloud).
+- Runtime and scheduling support for specialized hardware (GPUs, NUMA, RDMA, etc.)
+
+### Out of scope
+
+- Uses of the batch API resources as support for serving workloads (eg. backups, upgrades, migrations)
+- Proposals that reimplement the functionality of core kubernetes components (job-controller, kube-scheduler, cluster-autoscaler).
```

Comment:
I would say a replacement is in scope, which may result in reimplementing a controller, for example a v2 Job api.

Reply:
Ah, I was referring to forks that are supposed to run alongside.

Reply:
rephrased